lib/actions/ssh.ts

```diff
@@ -77,7 +77,7 @@ async function getContainerId(
 		// We need to execute a balena ps command on the device,
 		// and parse the output, looking for a specific
 		// container
-		const { child_process } = await import('mz');
+		const childProcess = await import('child_process');
```

Comment:
Strictly speaking this isn't `fs` but since the only method we use is `spawn` and it's not touched by mz( see https://github.com/normalize/mz/blob/2.7.0/child_process.js#L5-L6) I replaced this with the direct import so we could drop mz altogether